### ISSUE TITLE
FR:  Embedding a Group Link or Area Link into Another Excalidraw

### ISSUE THREAD

**Issue by Cabjhegas** [zsviczian/obsidian-excalidraw-plugin#1712]
**Is your feature request related to a problem? Please describe.**
My typical use case is: a large Excalidraw board used for brainstorming, planning, getting an overview, etc. This Excalidraw is not exported to SVG because it generates a very large file.
I need to take group links or area links from this large Excalidraw to bring into a simple MD file. If I publish this MD, the image won't work because it would need to be exported to SVG.

**Describe the solution you'd like**
It would be great if I could embed this group or area link in another Excalidraw file (just like it's possible in a simple MD file). This way, I could configure only this "sub-part Excalidraw" for SVG, and then embed this SVG in the simple MD file that will be published.

**Describe alternatives you've considered**
I tried using the crop tool, but it doesn't work for this case. To work around this, what I've been doing is, instead of the subpart Excalidraws being group links from the Big Excalidraw, the BIG Excalidraw is actually made up of several subpart Excalidraws.

**Additional context**
![image](https://github.com/zsviczian/obsidian-excalidraw-plugin/assets/26459442/571e0396-2733-403c-9eed-4da09a751f34)

![image](https://github.com/zsviczian/obsidian-excalidraw-plugin/assets/26459442/5ecece96-9bf9-45e9-a43f-2f43937a4f35)

![image](https://github.com/zsviczian/obsidian-excalidraw-plugin/assets/26459442/215eeddb-eb70-4b13-a918-b91e3b45ac9a)

**Comment by zsviczian** [comment]
yes, for now, my recommended solution is to deconstruct the selected elements into a separate file. That will be better from a performance perspective... but I also see the value of this requirement, and it is already on my backlog.

**Comment by Cabjhegas** [comment]
One way to make this alternative more practical would be to have the option to create a new Excalidraw based on the selection (and automatically embed it in the original position). This way, the deconstruction of the Big Excalidraw into subparts would be more seamless. :)
